java/src/jmri/configurexml/StoreXmlConfigAction.java

```diff
@@ -75,8 +75,8 @@ public static File getFileCustom(JFileChooser fileChooser) {
         // check for possible overwrite
         if (file.exists()) {
             int selectedValue = JOptionPane.showConfirmDialog(null,
-                    "File " + file.getName() + " already exists, overwrite it?",
-                    "Overwrite file?",
+                    rb.getString("FileOverwriteWarning1") + " " + file.getName() + " " + rb.getString("FileOverwriteWarning2"),
```

Comment:
Thanks for the hints, update is in PR #1028 
Advice to perform Ant Clean after working on packages and extends in another thread was also essential to silence IDE warnings